includes/tracks/events/class-wc-orders-tracking.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * WooCommerce Import Tracking
+ *
+ * @package WooCommerce\Tracks
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * This class adds actions to track usage of WooCommerce Orders.
+ */
+class WC_Orders_Tracking {
+	/**
+	 * Init tracking.
+	 */
+	public static function init() {
+		add_action( 'woocommerce_order_edit_status', array( __CLASS__, 'track_orders_edit_status_change' ), 10, 2 );
+		add_action( 'admin_init', array( __CLASS__, 'track_orders_view' ), 10 );
```

Comment:
I'd be more specific with this hook - perhaps either the `load-edit.php` action or even the `manage_shop_order_posts_columns` filter. The latter would allow you to remove the checks in `tracks_order_view()`.

Reply:
Thanks @jeffstieler, I was sure there had to be a better way.

>The latter would allow you to remove the checks in `tracks_order_view()`.

Can you explain what you mean here?

Reply:
> Can you explain what you mean here?

The `manage_shop_order_posts_columns` filter only fires on the Orders listing page, so you wouldn't need to check `SCRIPT_NAME` for a certain php file, and you don't need to check the post type.

Reply:
Oh right, of course. This fires once for every order on the page, however

Reply:
> This fires once for every order on the page, however

Ah, whoops. Probably best to use `load-edit.php` instead then. 👍